cmd/ipam/main.go

```diff
@@ -72,6 +72,7 @@ func main() {
 	cmd.Flags().IntVar(&options.ServerOpts.Port, "port", consts.IpamPort, "The port on which to listen for incoming gRPC requests.")
 	cmd.Flags().DurationVar(&options.ServerOpts.SyncFrequency, "interval", consts.SyncFrequency,
```

Comment:
I know it is not in this pr, but I noticed it in the manifest. Here, we have a name for the flag (interval) different from the const (sync frequency). I would use something like sync interval, also because the value is not a frequency but an interval